### DISCUSSION TITLE
✅ WOL in Tools

### DISCUSSION THREAD

**Discussion by hinrichd** [isontheline/pro.webssh.net#1252] [Ideas]
I would like to see wake on lan as feature under tools! 

Regards

**Comment by isontheline** [comment]
Hey @hinrichd 👋

No problem, please track this issue : https://github.com/isontheline/pro.webssh.net/issues/154

I will work on it soon

**Reply by isontheline** [reply]
Now available on the App Store

**Reply by hinrichd** [reply]
Just saw it in the release notes, great job! 
Thank You
